packages/cli/src/webhooks/webhook-helpers.ts

```diff
@@ -896,3 +892,21 @@ function evaluateResponseHeaders(context: WebhookExecutionContext): WebhookRespo
 
 	return headers;
 }
+/**
+ * Either return the original message, or a generic one if we don't want to surface the underlying cause.
+ *
+ * ONLY EXPORTED FOR TESTING.
+ *
+ * @param err the error being handled
+ */
+export function _privateGetWebhookErrorMessage(err: any, webhookType: 'Form' | 'Webhook'): string {
```

Comment:
Would `unknown` work here?

```suggestion
export function _privateGetWebhookErrorMessage(err: unknown, webhookType: 'Form' | 'Webhook'): string {
```